src/components/NavBar.js

```diff
@@ -58,59 +59,56 @@ class NavBar extends React.PureComponent {
             alt="North Shore Economic Vitally Partnership Logo"
           />
         </a>
-        <div style={{ width: '100%', marginLeft: '16px' }}>
+        <div style={{ marginLeft: '16px' }}>
           <h3 style={{ margin: '0px' }}>North Shore</h3>
           <h3 style={{ margin: '0px' }}>EVP</h3>
         </div>
         <nav>
           {isSignedIn(credentials) && (
             <ul>
-              <li
-                className={`${
-                  pathname === '/' ? 'nav-item-selected' : 'nav-item'
-                } nav-item-styling`}
-              >
-                <Link to="/">Dashboard</Link>
-              </li>
-              <li
-                className={`${
-                  pathname === '/' ? 'nav-item-selected' : 'nav-item'
-                } nav-item-styling`}
-              >
-                <Link to="/farms">Farms</Link>
-              </li>
-              <li
-                className={`${
-                  pathname === '/' ? 'nav-item-selected' : 'nav-item'
-                } nav-item-styling`}
-              >
-                <Link to="/supply">Supply</Link>
-              </li>
-
-              <li
-                className={`${
-                  pathname === '/' ? 'nav-item-selected' : 'nav-item'
-                } nav-item-styling`}
-                style={{
-                  borderRadius: '5px',
-                  border: '1px solid #4074B0',
-                  padding: '0px 18px'
-                }}
-              >
-                <Link to="/farm/asdf" style={{ color: '#4074B0' }}>
-                  New Farm
-                </Link>
-              </li>
-              {/* {isGeneralUser(credentials) && ()} */}
-              {/* isSubscriberUser(credentials) && (
+              <div className="nav-left-container">
+                <li
+                  className={`${
+                    pathname === '/' ? 'nav-item-selected' : 'nav-item'
```

Comment:
We're already using react-router-dom to create routes for our application. I think that the library also has NavLinks to handle setting an active CSS class for the page we're currently on. Would be a lot easier than using ternary ops.


https://reactrouter.com/web/api/NavLink
https://rdomanoel.medium.com/how-to-create-an-active-classname-navbar-using-react-and-react-router-70d890c9b58e

Reply:
nice catch myles!